File: 3541.py
from collections import Counter

class Solution:
    def maxFreqSum(self, s: str) -> int:
        vowels = set(['a', 'e', 'i', 'o', 'u'])
        vowel_max, consonant_max = 0, 0
        letters = Counter(s)

        for key, value in letters.items():
            if(key in vowels):
                vowel_max = max(vowel_max, value)
            else:
                consonant_max = max(consonant_max, value)

        return vowel_max + consonant_max

File: 3541.cpp
class Solution {
public:
    int maxFreqSum(string s) {
        unordered_set<char> vowels;
        unordered_map<char, int> count;
        int freq_v = 0, freq_c = 0;
        vowels.insert('a');
        vowels.insert('e');
        vowels.insert('i');
        vowels.insert('o');
        vowels.insert('u');

        for(int i = 0; i < s.size(); i++) {
            count[s[i]]++;
        }

        for(auto it = count.begin(); it != count.end(); it++) {
            if(vowels.find(it->first) != vowels.end()) {
                freq_v = fmax(freq_v, it->second);
            } else {
                freq_c = fmax(freq_c, it->second);
            }
        }
        return freq_v + freq_c;
    }
};